Dockerfile

```
@@ -0,0 +1,33 @@
+FROM node:16-alpine as base
+# Check https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine to understand why libc6-compat might be needed.
+RUN apk add --no-cache libc6-compat
+
+ARG PNPM_VERSION=7.1.1 \
+    APP \
+    # Next.js collects completely anonymous telemetry data about general usage.
+    # Learn more here: https://nextjs.org/telemetry
+    NEXT_TELEMETRY_DISABLED=1
+
+ENV APP=${APP} \
+    NEXT_TELEMETRY_DISABLED=${NEXT_TELEMETRY_DISABLED}
+
+WORKDIR /workspace/cfa_ui
+
+RUN corepack enable
+RUN corepack prepare pnpm@${PNPM_VERSION} --activate
```

Can't we combine these?
```suggestion
RUN corepack enable && \
    corepack prepare pnpm@${PNPM_VERSION} --activate
```